doc/collate1d.rst

```diff
@@ -154,9 +158,19 @@ command line or set the ``ignore_flux = True`` in the configuration file.
 
 Alternatively ``pypeit_collate_1d`` can perform flux calibration itself using archived sensitivity functions.
 To do so pass ``--flux`` to the command line or set ``flux = True`` in the configuration file.
-Doing so will overwrite any flux calibration data already in the spec1d files.
+if ``spec1d_output`` is set on the command line or in a configuration file the new spec1d will be written
```

Comment:
It seems that this is called `spec1d_outdir`

Reply:
Fixed